content/v3/git/refs.md

```diff
@@ -14,11 +14,26 @@ The `ref` in the URL must be formatted as `heads/branch`, not just `branch`. For
 
     GET /repos/:owner/:repo/git/refs/heads/skunkworkz/featureA
 
-### Response
-
 <%= headers 200 %>
 <%= json :ref %>
 
+If the ref doesn't exist in the repository, but existing refs start with ref
```

Comment:
Maybe some more emphasis on the `ref`, with some highlighting so that it's obvious we're talking about the path variable `ref`?